joomla/joomla-cms
# [4.0] Add tooltips for fields, field-groups list view text-search box #pbf19 #codepatch

### Summary of Changes
Add tooltips for fields, field-groups list view text-search box

### Testing Instructions
1. Apply patch
2. Verify if tooltip shown for search box for fields, field groups

### Expected result
Tooltip shown for search box for fields, field groups

### Actual result
Tooltip shown for search box for fields, field groups

### Documentation Changes Required
No
